### PR TITLE
[framework-bundle] Add Test for TranslationUpdateCommand

### PR DESCRIPTION
**Pull request by zerustech**
Added the test script as per the discussion in PR #15562

| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | n/a |
| License | MIT |
| Doc PR | n/a |

### CODE REVIEW

**Comment by zerustech** [comment]
ping @stof @aitboudad

File: src/Symfony/Bundle/FrameworkBundle/Tests/Command/TranslationUpdateCommandTest.php [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+    protected function tearDown()
+    {
+        $this->fs->remove($this->translationDir);
+    }
+
+    /**
+     * @return CommandTester
+     */
+    private function createCommandTester($container)
```
Comment (stof) [inline src/Symfony/Bundle/FrameworkBundle/Tests/Command/TranslationUpdateCommandTest.php]: you should typehint the argument

**Comment by stof** [comment]
The new test you are adding is broken

**Comment by zerustech** [comment]
@stof Fixed.

**Comment by fabpot** [comment]
Thank you @zerustech.

**Pull request merged**